workflows/sample/tasks/hifiasm_trio.wdl

```diff
@@ -0,0 +1,291 @@
+version 1.1
+
+#import "../../common/structs.wdl"
+
+import "https://raw.githubusercontent.com/ducatiMonster916/pb-human-wgs-workflow-wdl/main/workflows/common/structs.wdl"
+import "https://raw.githubusercontent.com/ducatiMonster916/pb-human-wgs-workflow-wdl/main/workflows/sample/tasks/hifiasm.wdl" as hifiasm
+
+
+task hifiasm_trio_assemble {
+  input {
+    Int threads = 48
+    String sample_name
+    String prefix = "~{sample_name}.asm"
+    String log_name = "hifiasm.log"
+
+    Array[File] movie_fasta
+    File parent1_yak
+    File parent2_yak
+    String pb_conda_image
+  }
+
+  Float multiplier = 2
+  Int disk_size = ceil(multiplier * size(movie_fasta, "GB")) + 20
+#  Int disk_size = 200
+  Int memory = threads * 3              #forces at least 3GB RAM/core, even if user overwrites threads
+
+  command <<<
+    echo requested disk_size =  ~{disk_size}
+    echo
+    source ~/.bashrc
+    conda activate hifiasm
+    echo "$(conda info)"
+
+    (hifiasm -o ~{prefix} -t ~{threads} -1 {parent1_yak} -2 {parent2_yak} ~{sep=" " movie_fasta} \
+    && (echo -e "hap1\t{params.parent1}\nhap2\t{params.parent2}" > ~{prefix}.key.txt) > ~{log_name} 2>&1
```

Comment:
Maybe I missed it, but it doesn't look like you're saving this key in the output.  This tells the user which parent is hap1 and which is hap2.

Reply:
Will check with @vsmalladi on this as this was a change he made.